docs/openapi.yaml

```diff
@@ -898,6 +898,58 @@ paths:
         400:
           description: invalid params
           content: {}
+  /export:
```

Comment:
It seems that this api path should begins with '/namespaces/{namespace}/models/{model}', e.g., /namespaces/{namespace}/models/{model}/export